tests/em/fdem/forward/test_FDEM_sources.py

```diff
@@ -68,13 +69,13 @@ def test_MagDipole(self):
     def test_MagDipoleSimpleFail(self):
 
         print('\ntesting MagDipole error handling')
-        with self.assertRaises(Exception):
-            FDEM.Src.MagDipole([], freq=self.freq, loc=np.r_[0., 0., 0.],
-                               orientation=np.r_[2., 3., 2.])
+
 
         with warnings.catch_warnings(record=True):
-            FDEM.Src.MagDipole([], freq=self.freq, loc=np.r_[0., 0., 0.],
-                               orientation=np.r_[1., 0., 0.])
+            FDEM.Src.MagDipole(
```

Comment:
good call, there is a typo in this test - I should be testing if a warning is thrown for non-axes aligned sources. they are not tested